src/detect-flowbits.c

```diff
@@ -212,13 +207,124 @@ int DetectFlowbitMatch (DetectEngineThreadCtx *det_ctx, Packet *p,
     return 0;
 }
 
-static int DetectFlowbitParse(const char *str, char *cmd, int cmd_len, char *name,
-    int name_len)
+static int DetectFlowbitParse(
+        DetectEngineCtx *de_ctx, const char *rawstr, DetectFlowbitsData **cdout)
 {
+    bool cmd_set = false;
+    bool name_set = false;
+    uint8_t cmd = 0;
+    char name[256] = "";
+    char *val = NULL;
+    DetectFlowbitsData *cd = NULL;
+    char copy[strlen(rawstr) + 1];
+    strlcpy(copy, rawstr, sizeof(copy));
```

Comment:
wonder if things would be simpler during parsing if we would copy only the non-`is_blank()` characters here. Then during tokenizing below we could ignore any spaces related logic.

Reply:
How do we handle "key<space>value" pairs then?
e.g. `set flowbit1`.